Rails 8.0 new generator with -T flag produces a Gemfile causing RuboCop and GHA lint trailing empty lines error

Hey, I'm running mentorship program and as part of it we are building a Rails 8.0 application from scratch. The initial step is to run rails new generator & commit the initial version to git & GitHub to showcase how all of these tools work out of the box. We run rails new with -T flag to skip minitest and system tests as we plan to add RSpec later as part of the course. After running the generator and pushing to GitHub, the default GHA linter fails with Rubocop error. 

It's not a big deal and developers with any experience with Rails will be able to fix this issue easily, but it's very confusing for new members in our community to have a GHA lint failure without adding any of the code by themselves. I feel the out of the box generated Rails app should come with all GHA checks being successful.

### Steps to reproduce:
1. run rails new generator with skip tests flag (`-T`):
```ruby
rails _8.0.1_ new app_name --database=postgresql -T
```
2. run Rubocop check:
```ruby
bin/rubocop -f github
```
and/or git commit initial version after generation:
```ruby
git commit -m "[1] - init: Rails 8.0.1, Ruby 3.4.1, PostgreSQL"
git push -u origin main
```

### Expected behavior:
Rubocop returns no error
After git push, all GHA checks are successful

### Actual behavior:
Rubocop fails with error:
```ruby
::error file=Gemfile,line=58,col=1::Layout/TrailingEmptyLines: 1 trailing blank lines detected.
```
After git push, GHA lint check fails:
```
Error: Layout/TrailingEmptyLines: 1 trailing blank lines detected.
Error: Process completed with exit code 1.
```
<img width="783" alt="Screenshot 2025-01-12 at 12 32 15" src="https://github.com/user-attachments/assets/3354e6bd-c94d-4018-a96e-6ebe8a1248b7" />

### System configuration:
**Rails version**: 8.0.1
**Ruby version**: 3.4.1
**OS**: macOS 14.0 (23A344)

--- a/railties/lib/rails/generators/rails/app/templates/Gemfile.tt
+++ b/railties/lib/rails/generators/rails/app/templates/Gemfile.tt
@@ -66,15 +66,15 @@ group :development, :test do
 <%- end -%>
 end
 <% end -%>
-
 <%- unless options.api? || options.skip_dev_gems? -%>
+
 group :development do
   # Use console on exceptions pages [https://github.com/rails/web-console]
   gem "web-console"
 end
 <%- end -%>
-
 <%- if depends_on_system_test? -%>
+
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem "capybara"

